main.tf

```diff
@@ -18,13 +18,22 @@ data "aws_subnets" "istio_public_subnet_ids" {
   }
 }
 
+data "aws_ec2_managed_prefix_list" "this" {
+  for_each = var.aws_managed_prefix_lists
+  name     = each.value
+}
+
 resource "helm_release" "istio-base" {
   name             = "istio-base"
   repository       = "https://istio-release.storage.googleapis.com/charts"
   chart            = "base"
   version          = var.istio_release_version
   namespace        = var.istio_release_namespace
   create_namespace = true
+  set {
+    name  = "defaultRevision"
+    value = "default"
+  }
 }
 
 resource "helm_release" "istio-discovery" {
```

Comment:
should be istio-discovery -> istio_discovery

Reply:
fixed